services/calculate.go

```diff
@@ -22,18 +22,23 @@ func CalculateProtocolScore(protocol string) (score int, message string) {
 	return
 }
 
+// GetCertificate returns the certificate associated with a host
+func GetCertificate(host string) *models.Cert {
+	return models.NewCert(host)
```

Comment:
I think of models being like conventional POJOs, so in that case it would be easier to have functions as getters and setters rather than using the keyword `New` in front of each exported function.

Reply:
You mean rename `NewCert` to `GetCert`